flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/functions/utils/ScalarSqlFunction.scala

```diff
@@ -39,22 +38,22 @@ import scala.collection.JavaConverters._
 /**
   * Calcite wrapper for user-defined scalar functions.
   *
-  * @param name           function name (used by SQL parser)
+  * @param identifier           function name (used by SQL parser)
   * @param displayName    name to be displayed in operator name
   * @param scalarFunction scalar function to be called
```

Comment:
Keep "function name" vertical alignment with other param docs.

Reply:
Please skip the first commit.